src/geocat/viz/util.py

```diff
@@ -556,6 +556,8 @@ def set_titles_and_labels(ax: typing.Union[matplotlib.axes.Axes,
     >>> |                   Axes                    |
     >>> |                                           |
 
+    Be aware that the `suptitle` functionality does not automaticallly render well for Cartopy plots. Recommended fixes are to iterate on your figure size until it does not extend as far past your plot OR to use string manipulation to add enough spaces to center your left or right title keyword argument.
```

Comment:
Should we add an admonition to the docs for this?

I'm not sure how good of practice it is, but conceivably we could also reference that issue for some longer form suggestions.

Reply:
Looking at it again, I think your original reply that he needed to use an appropriate figure size is the best solution. He had set his height to 11 inches, but if you use 4 it works just fine. That is common practice when dealing with Cartopy's vertical rendering spreading out to take up all of the space given to it by the figure. This now seems common enough that I don't think we need to link to the issue or have an extra annotation.

Here's another example: https://stackoverflow.com/questions/73826115/cartopy-too-much-vertical-space-between-subplots-even-with-tight-layout